centreon/src/Core/Dashboard/Domain/Model/Dashboard.php

```diff
@@ -54,18 +54,22 @@ class Dashboard
     public function __construct(
         protected readonly int $id,
         string $name,
-        string $description,
+        ?string $description,
```

Comment:
If it is nullable why adding it to the constructor ?

Reply:
IMO even if the property is nullable it remains a mandatory property of a dashboard